python/pyarrow/__init__.py

```diff
@@ -203,30 +202,49 @@ def show_versions():
 import pyarrow.types as types
 
 
-# deprecated filesystems
+# deprecated top-level access
+
 
 from pyarrow.filesystem import FileSystem as _FileSystem, LocalFileSystem as _LocalFileSystem
```

Comment:
I think we have flake8 disabled on this file.

Reply:
yeah, that's from the other PR, will fix